src/app/Plans/components/Wizard/helpers.tsx

```diff
@@ -691,6 +705,20 @@ export const useEditingPlanPrefillEffect = (
   };
 };
 
+export const getClonedPlanDefaultName = (
```

Comment:
I considered this, but generally I try to avoid function names that start with a verb which is not the action being done. A name that begins with `clone` makes me think at first glance that the function clones something. Of course, "this function clones the plan default name" is nonsense, so you'd quickly figure it out, but it makes me stop for a moment.

That said I'm sure this codebase is full of names like that, so it's not a big deal. This one is wordy so I'll go with your suggestion.

Reply:
Actually, I can avoid the verb confusion by just using `clonedPlanDefaultName` instead of `clonePlanDefaultName`! Will do. Here I am writing a paragraph about a one-character difference though 🤦